pybossa/view/projects.py

```diff
@@ -449,13 +449,15 @@ def clone_project(project, form):
 
     proj_dict['owner_id'] = current_user.id
     proj_dict['owners_ids'] = [current_user.id]
-    proj_dict['short_name'] = form['short_name']
     proj_dict['name'] = form['name']
-    replacement = 'pybossa.run("%s")' % proj_dict['short_name']
-    pybossa_re = "(pybossa\.run\(\s*)(['\"]\S+['\"]\))"
-    proj_dict['info']['task_presenter'] = re.sub(pybossa_re,
-                                                 replacement,
-                                                 proj_dict['info'].get('task_presenter', ''))
+
+    task_presenter = proj_dict['info'].get('task_presenter', '')
+    for quoted_part in re.findall(r"[\'\"](.*?)[\"\']", task_presenter):
+        task_presenter = task_presenter.replace(quoted_part, quoted_part.replace(proj_dict['short_name'], form['short_name']))
```

Comment:
You might be able to use capture groups and back references here - https://docs.python.org/2/library/re.html#re.sub

Reply:
As discussed will just match  when the short name appears in "(name)" or '(name) ' or /(name)/